action.php

```diff
@@ -72,24 +73,41 @@ public function convert(Doku_Event $event) {
         // it's ours, no one else's
         $event->preventDefault();
 
-        // prepare cache and its dependencies
-        $depends = array();
-        $cache = $this->prepareCache($depends);
+        $generateNewPdf = false;
+        if ($REV || $DATE_AT) {
+            $tempFilename = tempnam('/tmp', 'dw2pdf_');
```

Comment:
/tmp may not be writable on some systems (shared webhosts) or may not exist (windows). Better use DokuWiki's temporary directory via `$conf['tmpdir']`

Reply:
implemented. Though `tempnam` should also fix this itself. The documentation states:
>  If the directory does not exist or is not writable, tempnam() may generate a file in the system's temporary directory, and return the full path to that file, including its name.